src/is-prerelease.js

```diff
@@ -0,0 +1,8 @@
+export default function isPrerelease({ type, main, prerelease }) {
+    if (prerelease === false) {
+        return false;
+    }
+    return (
```

Comment:
This seems a little complicated, could you please split these into a chain of if-else with "comment headers" describing how a plugin might "land" in each of these branches?

Also, please add some unit tests to demonstrate the same ideas (and so codecov stays at 100% 😊)